### DISCUSSION TITLE
How to set working dir running web application

### DISCUSSION THREAD

**Discussion by pomcho555** [buildpacks/community#90] [Q&A]
## Environment
- OSX 10.14.2
- python3.8
- fastapi==0.63.0
- pack: 0.17.0+git-d9cb4e7.build-2045

## Directory structure

```
my-app/
├── README.md
├── requirements.txt
├── app
│   ├── main.py
└── tests
```

## QUESTION

I am running a web app through FastAPI and Unicorn under `app` directory. For the sake of using `buildpacks pack`, I wrote a following one liner command. But it didn't work...
I don't know how to set the working directory as the app launched. On docker, we can set `WORKDIR` option for that. I need some kind of similar way.

```sh
pack build sample-app --builder gcr.io/buildpacks/builder:v1 --env CNB_APP_DIR="/app"\
 --env GOOGLE_ENTRYPOINT="uvicorn main:app"
```

**Comment by natalieparellano** [comment]
For posterity, `pack build` now supports `--workspace` flag: `Location at which to mount the app dir in the build image`
